python/google/protobuf/json_format.py

```diff
@@ -583,6 +587,11 @@ def _ConvertFieldValuePair(self, js, message):
             message.Extensions[field] = _ConvertScalarFieldValue(value, field)
           else:
             setattr(message, field.name, _ConvertScalarFieldValue(value, field))
+      except _UnknownEnumStringValueError as e:
```

Comment:
This implementation gives up early if there is an unknown enum value in an array of enum values. This is not good.